### PR TITLE
[SELS-TASK] Integration of Quiz Result page, plus minor upgrade of other pages

### PR DESCRIPTION
**Pull request by NiloJr-sun**
## Asana Link 
- [Integration of Quiz results](https://app.asana.com/0/1202703680910766/1202869845952302/f)
## Commands
```
- cd backend 
- php artisan serve
- cd ../frontend
- npm start
```
##  Difinition of done
-  [x] Integration of User dashboard header, now determines time (Morning, afternoon, evening)
-  [x] Integration of User quiz page
-  [x] Integration of Categories page UI, bootstrap wise
-  [x] Integration of Quiz results
-  [x] Clean up of unused hooks from Admin dashboard
-  [x] Addition of Quiz result to Routes

## Notes
-
## Screenshots
image
image2
image3
image4
image5
image6

### CODE REVIEW

File: frontend/src/components/admindashboard/addcategroy.js [hunk 1]
```diff
@@ -1,46 +1,40 @@
...
 import Cookies from 'js-cookie';
+import { ToastContainer, toast } from 'react-toastify';
+import { useLocation } from 'react-router-dom';
+
 
 const REG_URL = '/categories';
 const Addcategory = () => {
-    const errRef = useRef();
-    const sucRef = useRef();
+    const loc = useLocation();
```
Comment (ejanton-sun) [inline frontend/src/components/admindashboard/addcategroy.js]:
```suggestion
    const location = useLocation();
```

File: frontend/src/components/admindashboard/admindashboard.js [hunk 2]
```diff
@@ -16,13 +17,22 @@ const Admindashboard = () => {
 
   const fetchData = (pageNumber) => {
     axios
-      .get(REG_URL + '?page=' + pageNumber)
+      .get(CAT_URL + '?page=' + pageNumber)
```
Comment (ejanton-sun) [inline frontend/src/components/admindashboard/admindashboard.js]:
```suggestion
      .get(`${CAT_URL}?page=${pageNumber}`)
```

**Review by ejanton-sun — CHANGES_REQUESTED** [review]

File: frontend/src/components/admindashboard/admindashboard.js (hunk 2 above)
Comment (ejanton-sun) [reply frontend/src/components/admindashboard/admindashboard.js]: Pls apply this to all axios routes.
Reference: [ Template literals (Template strings)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

File: frontend/src/components/admindashboard/addcategroy.js (hunk 1 above)
Comment (NiloJr-sun) [reply frontend/src/components/admindashboard/addcategroy.js]: I think I should delete const loc sir, I just noticed that it is unused.

**Review by ejanton-sun — APPROVED** [review]
# Approve

**Pull request merged**